tests_metricflow/generate_snapshots.py

```diff
@@ -32,6 +32,10 @@
         "engine_url": trino://...",
         "engine_password": "..."
     },
+    "clickhouse": {
```

Comment:
There's a 1pass entry that stores these creds for easy access, so it would be great if we could update that too!